### PR TITLE
Avoid deadlock between RequestProcessor and FileChooserBuilder

### PR DESCRIPTION
**Pull request by DrSchranz**
Trying to solve issue #32.
Extracted FileChooserBuilder.showMultiDialog() from the logic passed to the RequestProcessor, and wrapped selectFiles() call with SwingUtilities.invokeLaker() to not fire showMultiDialog() in EDT.

### CODE REVIEW

File: src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java [hunk 1]
```diff
@@ -1445,22 +1450,15 @@ public void run() {
         "BacklogIssuePanel.message.uploading.attachments=Uploading files"
     })
     private void selectFiles() {
-        // attachment
-        RP.post(new Runnable() {
 
+        class AttachmentUploader implements Runnable {
```
Comment (junichi11) [inline src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java]: Please use private static class.

File: src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java [hunk 2]
```diff
@@ -1445,22 +1450,15 @@ public void run() {
...
+            
+            File[] attachments;
+            
             @Override
             public void run() {
+     
                 submitHeaderButton.setEnabled(false);
-                // TODO add FileFilter
```
Comment (junichi11) [inline src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java]: Why did you remove this comment?

**Comment by junichi11** [comment]
@DrSchranz Thanks for PR! I left comments in your changes.

**Comment by junichi11** [comment]
If you can fix them, please change something then commit and push them. You need not to create a new PR :)

File: src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java (hunk 1 above)
Comment (DrSchranz) [inline src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java]: I thought that local class would be similar to the anonymous one, and was created because i was unable to put `File[] attachments` in `selectFiles()` scope and use it from the anonymous `Runnable.run()` inner scope.
I like the private static nested class, but static would require a way to share current IssuePanel instance properties?

Anyway now I see that a `final File[] attachments` in `selectFiles()` scope will allow the access from the anonymous Runnable passed to RP. So maybe the change doesn't need to declare a class and just extract `File[] attachments = FileChoseBuilder ... ` outside the `Runnable.run()`. 
( I need to pay more attention to the clever netbeans! )

I am somewhat embarrassed because I feel lost between my lack of knowledge and OOP subjectivity.

Probably you could address this fix better, and because I'm not sure how to now revert this commits (Revert the pushed commits can affect Github badly, or leave an ugly history right?).
Thanks.

File: src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java (hunk 2 above)
Comment (DrSchranz) [inline src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java]: Now I'm quite embarrassed, at start, before I decided to try create PRs, I was just messing with the code trying to run the plugin in my netbeans, and removed it accidentally. I'm sorry need to check the diffs better.

**Comment by DrSchranz** [comment]
I'll try to commit commented changes in my branch. But I'm afraid that commit history will be ugly in your project if my failed commits are showed after merge?. So close the PR If you address this fix more properly.

Thanks for your time and chances!

File: src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java (hunk 1 above)
Comment (junichi11) [inline src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java]: I'm sorry. This comment is my mistake X(

**Comment by junichi11** [comment]
@DrSchranz Thanks a lot!
